x-pack/plugins/monitoring/public/components/kibana/instances/instances.js

```diff
@@ -0,0 +1,168 @@
+import { ClusterStatus } from '../cluster_status';
+import { EuiMonitoringTable } from '../../table';
+import { KibanaStatusIcon } from '../status_icon';
+import { formatMetric, formatNumber } from '../../../lib/format_number';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+const getColumns = (kbnUrl, scope) => {
+  const columns = [
+    {
+      name: i18n.translate('xpack.monitoring.kibana.listing.nameColumnTitle', {
+        defaultMessage: 'Name'
+      }),
+      field: 'name',
+      render: (name, kibana) => (
+        <EuiLink
+          onClick={() => {
+            scope.$evalAsync(() => {
+              kbnUrl.changePath(`/kibana/instances/${kibana.kibana.uuid}`);
+            });
+          }}
+          data-test-subj={`kibanaLink-${name}`}
+        >
+          { name }
+        </EuiLink>
+      )
+    },
+    {
+      name: i18n.translate('xpack.monitoring.kibana.listing.statusColumnTitle', {
+        defaultMessage: 'Status'
+      }),
+      field: 'status',
+      render: (status, kibana) => (
+        <div
+          title={`Instance status: ${status}`}
+          className="monTableCell__status"
+        >
+          <KibanaStatusIcon status={status} availability={kibana.availability} />&nbsp;
+          { !kibana.availability ? (
+            <FormattedMessage
+              id="xpack.monitoring.kibana.listing.instanceStatus.offlineLabel"
+              defaultMessage="Offline"
+            />
+          ) : capitalize(status) }
+        </div>
+      )
+    },
+    {
+      name: i18n.translate('xpack.monitoring.kibana.listing.loadAverageColumnTitle', {
+        defaultMessage: 'Load Average'
+      }),
+      field: 'os.load.1m',
+      render: value => (
+        <span>
+          {formatMetric(value, '0.00')}
+        </span>
+      )
+    },
+    {
+      name: i18n.translate('xpack.monitoring.kibana.listing.memorySizeColumnTitle', {
+        defaultMessage: 'Memory Size'
+      }),
+      field: 'process.memory.resident_set_size_in_bytes',
+      render: value => (
+        <span>
+          {formatNumber(value, 'byte')}
+        </span>
+      )
+    },
+    {
+      name: i18n.translate('xpack.monitoring.kibana.listing.requestsColumnTitle', {
+        defaultMessage: 'Requests'
+      }),
+      field: 'requests.total',
+      render: value => (
+        <span>
+          {formatNumber(value, 'int_commas')}
+        </span>
+      )
+    },
+    {
+      name: i18n.translate('xpack.monitoring.kibana.listing.responseTimeColumnTitle', {
+        defaultMessage: 'Response Times'
+      }),
+      // It is possible this does not exist through MB collection
+      field: 'response_times.average',
+      render: (value, kibana) => {
+        if (!value) {
+          return null;
+        }
+
+        return (
+          <div>
+            <div className="monTableCell__splitNumber">
+              { value && (formatNumber(value, 'int_commas') + ' ms avg') }
```

Comment:
@chrisronline I don't think this check is necessary, since you're already doing it on line 97.

Also, wouldn't `0` be a valid value? But, since it's falsey it will `return null;` on line 98